### PR TITLE
feat(FEC-10405): Casting with Airplay

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Configure airplay plugin when running on Safari.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/common/utils/setup-helpers.js [hunk 1]
```diff
@@ -638,6 +639,22 @@ function maybeSetCapabilitiesForIos(options: KPOptionsObject): void {
...
 }
 
+/**
+ * Configures airplay plugin in case we're running in Safari browser.
+ * @private
+ * @param {KPOptionsObject} options - kaltura player options
+ * @returns {void}
+ */
+function maybeConfigureAirplay(options: KPOptionsObject): void {
+  if (Env.isSafari) {
```
Comment (Yuvalke) [inline src/common/utils/setup-helpers.js:649]: Please add !Env.isSmartTV since LG2 recognize and safari browser

**Review by Yuvalke — CHANGES_REQUESTED** [review]

File: src/common/utils/setup-helpers.js (hunk 1 above)
Comment (OrenMe) [reply src/common/utils/setup-helpers.js:649]: so it makes more sense to check if ios or macOS then to check !smartTV in this context

**Review by OrenMe — COMMENTED** [review]
@dan-ziv I think we don't need this if we are going with explicit plugin addition via studio, right?

File: src/common/utils/setup-helpers.js (hunk 1 above)
Comment (dan-ziv) [reply src/common/utils/setup-helpers.js:649]: This PR is irrelevant since we will configure it via Studio

**Pull request closed without merging**